src/ansible_creator/resources/common/role/roles/run/README.md.j2

```diff
@@ -0,0 +1,57 @@
+{{ namespace|capitalize }}.{{ collection_name|capitalize }} Run Role
+========================
+
```

Comment:
If the role is going in as a common resource, the collection project should use it rather than have it's own collection.